app/views/providers/contact.html.erb

```diff
@@ -55,6 +55,13 @@
         <%= f.text_field field, options.merge(class: 'govuk-input govuk-!-width-two-thirds') %>
       <% end %>
 
+      <% if @provider.provider_type == "lead_school" %>
+        <%= render "shared/form_field",
+          form: f, field: :urn, label: "URN (Unique Reference Number)" do |field, options| %>
+          <%= f.text_field field, options.merge(class: 'govuk-input govuk-input--width-10') %>
+        <% end %>
+      <% end %>
```

Comment:
Can we put URN below the UKPRN text box please? It’s not your fault, I think the Trello ticket was slightly wrong.

Reply:
Of course, done! 👍